src/caret_analyze/plot/plot_facade.py

```diff
@@ -128,15 +126,14 @@ def create_period_timeseries_plot(
 
     @staticmethod
     def create_frequency_timeseries_plot(
-        *target_objects: TimeSeriesTypes
+        *target_objects: CallbackBase | Communication
```

Comment:
This API should accept `Publisher`, `Subscription` as well.
When `Publisher` is set: This API outputs the freq of the published topic
When `Subscription` is set: This API outputs the freq of the subscription callback

Reply:
fixed in https://github.com/tier4/caret_analyze/pull/463/commits/830035f931d3315a54f495a053d4962bc17032f6